podpac/core/data/ogc.py

```diff
@@ -4,373 +4,264 @@
-
-        timedomain = capabilities.find("wcs:temporaldomain")
-        if timedomain is None:
-            return Coordinates(
-                [
-                    UniformCoordinates1d(top, bottom, size=size[1], name="lat"),
-                    UniformCoordinates1d(left, right, size=size[0], name="lon"),
-                ]
-            )
-
-        date_re = re.compile("[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}")
-        times = str(timedomain).replace("<gml:timeposition>", "").replace("</gml:timeposition>", "").split("\n")
-        times = np.array([t for t in times if date_re.match(t)], np.datetime64)
-
-        if len(times) == 0:
-            return Coordinates(
-                [
-                    UniformCoordinates1d(top, bottom, size=size[1], name="lat"),
-                    UniformCoordinates1d(left, right, size=size[0], name="lon"),
-                ]
-            )
-
-        return Coordinates(
-            [
-                ArrayCoordinates1d(times, name="time"),
-                UniformCoordinates1d(top, bottom, size=size[1], name="lat"),
-                UniformCoordinates1d(left, right, size=size[0], name="lon"),
-            ]
-        )
+            return super()._eval(coordinates, output=output, _selector=selector)
 
-    @common_doc(COMMON_DATA_DOC)
-    def get_coordinates(self):
-        """{get_coordinates}
+        # for uniform stacked, unstack to use the requested coordinates (the WCS server will interpolate)
+        if "lat_lon" in coordinates.dims and coordinates["lat"].is_uniform and coordinates["lon"].is_uniform:
 
-        Notes
-        ------
-        This is a little tricky and doesn't fit into the usual PODPAC method, as the service is actually doing the
-        data wrangling for us...
-        """
+            def selector(rsc, rsci, coordinates):
+                unstacked = coordinates.unstack()
+                return unstacked, tuple(slice(None) for dim in unstacked)
 
-        # TODO update so that we don't rely on _requested_coordinates if possible
-        if not self._requested_coordinates:
-            return self.wcs_coordinates
-
-        cs = []
-        for dim in self.wcs_coordinates.dims:
-            if dim in self._requested_coordinates.dims:
-                c = self._requested_coordinates[dim]
-                if c.size == 1:
-                    cs.append(ArrayCoordinates1d(c.coordinates[0], name=dim))
-                elif isinstance(c, UniformCoordinates1d):
-                    cs.append(UniformCoordinates1d(c.bounds[0], c.bounds[1], abs(c.step), name=dim))
-                else:
-                    # TODO: generalize/fix this
-                    # WCS calls require a regular grid, could (otherwise we have to do multiple WCS calls)
-                    cs.append(UniformCoordinates1d(c.bounds[0], c.bounds[1], size=c.size, name=dim))
+            udata = super()._eval(coordinates, output=None, _selector=selector)
```

Comment:
The remote server **may** not support the user's crs. Probably need to transform the coordinates to support crs's before this step.

Reply:
Also, did you drop unsupported coordinates before this step?

Reply:
The coordinates are transformed by the DataSource._eval, **but** it is better to transform them here in order to select the correct case!

Reply:
Dims are dropped later, also.